Source/Mod/Patches/KeyboardHandler.cs

```diff
@@ -0,0 +1,225 @@
+﻿using System.Diagnostics;
+using System.Runtime.InteropServices;
+
+namespace Celeste64.Mod;
+
+class KeyboardHandler
+{
+	public static readonly Dictionary<string, string> KeyValues = new Dictionary<string, string>
```

Comment:
It might be useful if this was broken out into a couple dictionaries, instead of one big one, so it could be a bit more flexible. Like one group for letters, one for numbers, one for special characters, and one for keypad characters. And then maybe have some properties that aggregate them.
This could enable us to possibly prevent special characters or prevent numbers on the onscreen keyboard, or make it numbers only, which seems like it could be useful.